_posts/2017-12-05-december.html

```diff
@@ -11,16 +11,15 @@
     {% comment %} ==== TALK / PRESENTATION - SLOT #1 ====     {% endcomment %}
 
     {% comment %} TITLE OF YOUR TALK:                         {% endcomment %}
-                  <h4>Free Slot</h4>
+                  <h4>Recruting, Recruter, and how to find Developers</h4>
```

Comment:
Take note, though, that we are missing `i`s after the `u`s here (*to recruit*).

Reply:
thx for the hint, fixed it